cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -0,0 +1,543 @@
+|              |                        |
+|              |   field: material.code |
+|              |   operator: IN         |
+|              |   value:               |
+|              |    -­ GOLD             |
+|              |    - LEATHER           |
++--------------+------------------------+
+
+Number
+______
++--------------+------------------------+
+| Operator     | - <                    |
+|              | - <=                   |
+|              | - =                    |
+|              | - >                    |
+|              | - >=                   |
+|              | - EMPTY                |
++--------------+------------------------+
+| Value        | Number. If operator    |
+|              | is empty, values       |
+|              | information are        |
+|              | ignored.               |
++--------------+------------------------+
+| Example      | .. code-block:: yaml   |
+|              |                        |
+|              |   field: min_age       |
+|              |   operator: =          |
+|              |   value: 12            |
++--------------+------------------------+
+
+Date
+____
++--------------+------------------------+
+| Operator     | - <                    |
+|              | - =                    |
+|              | - >                    |
+|              | - BETWEEN              |
+|              | - NOT BETWEEN          |
+|              | - EMPTY                |
++--------------+------------------------+
+| Value        | Format date :          |
+|              | yyyy-mm-dd. If         |
+|              | operator is empty,     |
+|              | Values information     |
+|              | are ignored.           |
++--------------+------------------------+
+| Example      | .. code-block:: yaml   |
+|              |                        |
+|              |   field: fix_date      |
+|              |   operator: >          |
+|              |   value: 2016-05-12    |
++--------------+------------------------+
+
+Price
+_____
++--------------+------------------------+
+| Operator     | - <                    |
+|              | - <=                   |
+|              | - =                    |
+|              | - >                    |
+|              | - >=                   |
+|              | - EMPTY                |
++--------------+------------------------+
+| Value        | Numeric value and      |
+|              | currency code.         |
+|              | Dot "." is the decimal |
+|              | separator. No space    |
+|              | between thousands.     |
+|              | If operator is empty,  |
+|              | values information     |
+|              | are ignored.           |
++--------------+------------------------+
+| Example      | .. code-block:: yaml   |
+|              |                        |
+|              |   field: basic_price   |
+|              |   operator: <=         |
+|              |   value: 12 EUR        |
```

Comment:
Here's the format to use for price attributes (the one described do not work):
                field: price
                operator: =
                value: 
                    data: 12
                    currency: EUR